Bottom: ac9cd60674931f9459d1938e9e58880088abc96b
Top:    c78b5b160bb4a6fbe6b21a3ce6bfb1a3c2c52cda
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-25 18:30:13 +0100

draw.c: Initialise integer counters with 0, not "false".

For the "doing_assy" flag, fix its type to "bool" rather than "int".


---

diff --git a/src/draw.c b/src/draw.c
index 03724f5..a763d3c 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -77,10 +77,10 @@ FloatPolyType, *FloatPolyTypePtr;
  */
 static BoxType Block;
 static bool Gathering = true;
-static int Erasing = false;
+static int Erasing = 0;
 
-static int doing_pinout = false;
-static int doing_assy = false;
+static int doing_pinout = 0;
+static bool doing_assy = false;
 static const BoxType *clip_box = NULL;
 
 /* ---------------------------------------------------------------------------
